package.json

```diff
@@ -30,7 +30,9 @@
     "test:setup": "node tools/config.js",
     "pretest:saucelabs": "lerna run --parallel pretest",
     "test:saucelabs": "karma start config/karma.saucelabs.js --single-run",
-    "docgen": "node scripts/docgen/generate-docs.js --api js; node scripts/docgen/generate-docs.js --api node"
+    "docgen": "node scripts/docgen/generate-docs.js --api js; node scripts/docgen/generate-docs.js --api node",
```

Comment:
no need to repeat the command -
`
"docgen": "npm run docgen:js; npm run docgen:node"
`

Reply:
Oh right, done.